src/plugins/intel_gpu/src/graph/primitive_inst.cpp

```diff
@@ -1068,27 +1068,22 @@ void primitive_inst::convert_args(const kernel_arguments_data& args, kernel_argu
 }
 
 int32_t primitive_inst::get_index_in_deps(memory::cptr arg) const {
-    uint32_t idx = 0;
-
-    for (idx = 0; idx < _deps.size(); ++idx) {
+    for (uint32_t idx = 0; idx < _deps.size(); ++idx) {
         if (arg == _deps[idx]->_outputs[0])
-            break;
+            return idx;
     }
 
-    if (idx == _deps.size())
-        std::cout << "[get_index_in_deps]: not found" << std::endl;
-
-    return (idx == _deps.size()) ? -1 : idx;
+    IE_THROW() << "[get_index_in_deps]: not found in _deps";
```

Comment:
https://github.com/openvinotoolkit/openvino/pull/13801#discussion_r1021112909
Changed to throw an exception. Thank you.